plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -391,6 +392,22 @@ class ServicesTable extends React.Component {
     );
   }
 
+  renderInstances(prop, service) {
+    const instancesCount = service.getInstancesCount();
+    const tasksRunning = service.getTaskCount();
+    const overview = tasksRunning === instancesCount
+      ? ` ${tasksRunning}`
+      : ` ${tasksRunning}/${instancesCount}`;
+
+    const content = isNaN(instancesCount) ? "\u2014" : overview;
```

Comment:
If you want to be sure `instancesCount` is really a number you should also check `typeof` as you otherwise might get a false positive (e.g., `isNaN("")`).

Reply:
Agree, I have changed to make sure is only `integer`. Thanks